Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -131,6 +145,21 @@ RDKIT_FILEPARSERS_EXPORT void MolToMolFile(
     const ROMol &mol, const std::string &fName, bool includeStereo = true,
     int confId = -1, bool kekulize = true, bool forceV3000 = false);
 
+// \brief Writes a molecule to an MDL V3000 mol file
+/*!
+ *   \param mol           - the molecule in question
+ *   \param fName         - the name of the file to use
+ *   \param includeStereo - toggles inclusion of stereochemistry information
+ *   \param confId        - selects the conformer to be used
+ *   \param kekulize      - triggers kekulization of the molecule before it is
+ * written
+ */
+inline void MolToV3KMolFile(const ROMol &mol, const std::string &fName,
+                            bool includeStereo = true, int confId = -1,
+                            bool kekulize = true) {
+  MolToMolFile(mol, fName, includeStereo, confId, kekulize);
```

Comment:
I think you forgot passing `forceV3000 = true`

Reply:
oh my...